Source/Core/VideoBackends/OGL/Render.cpp

```diff
@@ -644,6 +646,8 @@ Renderer::Renderer()
 	glDisable(GL_STENCIL_TEST);
 	glEnable(GL_SCISSOR_TEST);
 
+	if (g_ActiveConfig.backend_info.bSupportsClipControl)
+		glClipControl(GL_LOWER_LEFT, GL_ZERO_TO_ONE);
```

Comment:
Right, we still write with glClear. I only wanted to rewrite this with a draw call :/